tools/webpack/blocks.js

```diff
@@ -35,6 +35,8 @@ const prefixFunctions = [
 	'wp_get_global_settings',
 ];
 
+const suffixClasses = [ 'WP_Navigation_Block_Renderer' ];
```

Comment:
oh another list to maintain?

Reply:
I don't consider this a blocker for this PR but if we can avoid the list, it would be an improvement.

Reply:
Agreed. Ship this and then follow up to make this dynamic (unless we can easily do the dynamic bit here).